.github/workflows/publish.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - uses: dschep/install-poetry-action@v1.2
+    - uses: snok/install-poetry@v1.1.1
```

Comment:
Just for history.
The change is because the action `dschep` is not maintained anymore: https://github.com/dschep/install-poetry-action

Reply:
Thanks Toto, I mentioned it also here because renovate doesn't catch this:
https://github.com/canonical-web-and-design/canonicalwebteam.store-api/pull/47#issuecomment-729552879